glib/src/value.rs

```diff
@@ -205,6 +205,18 @@ pub unsafe trait FromValue<'a>: Sized {
     unsafe fn from_value(value: &'a Value) -> Self;
 }
 
+/// Similar to `FromValue` but works for owned `Value.
+pub unsafe trait FromOwnedValue: Sized {
```

Comment:
```suggestion
pub unsafe trait FromOwnedValue: for<'a> FromValue<'a> {
```

Then you can remove the `Checker` probably